src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.kuali.kfs.krad.exception.ValidationException;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
+
+public class ConcurRequestExtractFileValidationServiceImplTest {
+    private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
+    
+    @Before
+    public void setUp() throws Exception {
+        Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
+        concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        concurRequestExtractFileValidationService = null;
+    }
+    
+    @Test
+    public void testHeaderAmountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
```

Comment:
This comment pertains to this concern and the next three. These four tests are in pairs with match test first followed by mis-match test next.   I think the issue here is the assert processing and I will rework that to make it clearer what is happening.